util/php-dictionary-gen/README.md

```diff
@@ -0,0 +1,3 @@
+Use gen.sh to generate the dictionary files. The script will download the latest version of the PHP source code and generate the dictionary files from it. It will search the function on Github to count the number of uses. *Important*: First set GITHUB_TOKEN in gen.sh, bot.py requires GITHUB_TOKEN env variable.
```

Comment:
Prompt the token from the user and export the variable for use by the other scripts. Then you don't need the extra instructions here.

Reply:
Just added one additional check.